docs/howto/use-extensions/list-extensions.rst

```diff
@@ -0,0 +1,11 @@
+.. _list-extensions:
+
+List extensions
+===============
+
+Different versions of Snapcraft and its various cores support different extensions. To
+list all extensions supported by the installed version of Snapcraft, run:
+
+.. code-block:: bash
+
+    snapcraft extensions
```

Comment:
I believe the long-form version of the command is preferred.

```suggestion
    snapcraft list-extensions
```

Reply:
9d396a3